### PR TITLE
Adapt code style and do some improvment

### PR DESCRIPTION
**Pull request by rakati**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -50,3 +50,7 @@ modules.order
 Module.symvers
 Mkfile.old
 dkms.conf
+
+# execution files
+*.out
+*.exe
```
Comment (rakati) [inline .gitignore:56]: why we need this?

Comment (rakati) [reply .gitignore:56]: bla bla

File: helper.c [hunk 2]
```diff
@@ -1,77 +1,63 @@
...
-		default:
-			return (-1);
+	case 'c':
+		counter = _putchar(va_arg(args, int));
+		break;
+	case 's':
+		counter = print_str(va_arg(args, char *));
+		break;
+	case '%':
+		counter = _putchar('%');
```
Comment (rakati) [inline helper.c]: handle putchar error case

File: helper.c [hunk 3]
```diff
@@ -1,77 +1,63 @@
...
+		break;
+	default:
+		_putchar('%');
```
Comment (rakati) [inline helper.c]: add to counter

**Review by amine-zzr — APPROVED** [review]

**Comment by amine-zzr** [comment]
everything is well organized.

**Pull request merged**